lxd/instances_post.go

```diff
@@ -720,6 +720,11 @@ func createFromBackup(s *state.State, r *http.Request, projectName string, data
 		return response.BadRequest(err)
 	}
 
+	sysinfo, err := cluster.LocalSysInfo()
+	if err != nil {
+		return response.InternalError(err)
```

Comment:
I suggest using response.SmartError() here when your not generating the error yourself, as it could return different types of error now or in the future.